stories/topics/ref-gcp-populate-restore-file.adoc

```diff
@@ -21,26 +20,26 @@ The following parameters must be removed:
 Provide values for the following parameters:
 
 * `gcp_existing_vpc` must be set to `false`.
-* `cloud_credentials_path` is the absolute path toward your credentials.
-* `gcp_deployment_name` is the name of the deployment you want to back up.
-* `gcp_restored_deployment_name` isthe name under which the deployment must be restored.
-* `gcp_bucket_backup_name` is the bucket you used for the backup.
-* `gcp_bucket_backup_name` is the name you used for backup.
+* `cloud_credentials_path` is the path for your Google Cloud service account credentials file.
+* `gcp_deployment_name` is the name of the AAP deployment manager deployment you want to back up.
+* `gcp_restored_deployment_name` is the name under which the deployment must be restored. A new deployment will be created with this name. A deployment must not already exist with this name.
+* `gcp_bucket_backup_name` is the bucket name you used for the backup.
 * `gcp_compute_region` is the region where the backup was taken.
+This can be retrieved by checking the Deployments config in Deployment Manager.
 * `gcp_compute_zone` is the zone where the backup was taken.
+This can be retrieved by checking the Deployments config in Deployment Manager.
 
+// .For an existing VPC
 
-.For an existing VPC
+// If you want to restore using an existing VPC, you must set the parameters shown above.
```

Comment:
You've said twice that existing VPC networks can't be used.

Reply:
I believe this is jut commenting it out for now because we will most likely add the function in the next release